.github/actions/configure-guides-step/action.yaml

```diff
@@ -24,7 +24,7 @@ inputs:
 
 runs:
   using: 'docker'
-  image: 'docker://ghcr.io/typo3-documentation/render-guides:main'
+  image: 'docker://ghcr.io/typo3-documentation/render-guides:stable'
```

Comment:
Think this should be kept on main for now. As you want to have the latest changes for testing?